.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve.
+labels: bug
+---
+
+**Description**
+
+A clear and concise description of what the bug is.
+
+* Tarantool version (`tarantool --version`):
+* OS: [e.g. Linux/MacOS/FreeBSD/OpenBSD]
+* OS Version: [e.g. Debian 11, FreeBSD 11]
+* Architecture: [amd64, arm]
+* Log output (or attach a file with output) and backtrace
+
+**Steps to reproduce**
+
+1. Go to '...'
+1. Click on '....'
+1. Scroll down to '....'
```

Comment:
I would give something more tarantool specific: those steps looks like we develop a GUI application. E.g.:

----

1. Run two tarantool instances iteractively.
2. Type '...' in console 1.
3. Type '...' in console 2.

Or:

Reproducer script:

```lua
#!/usr/bin/env tarantool

<your code here>
```

How to run:

Just type `tarantool test.lua`.

----

Real-world examples: #5559, #4294, #4114, #4106.

Reply:
Done.